tests/integration/edge-handler.test.ts

```diff
@@ -261,27 +261,27 @@ describe("aborts middleware execution when the matcher conditions don't match th
 
     ctx.cleanup?.push(() => origin.stop())
 
-    for (const path of ['/hello', '/en/hello', '/nl-NL/hello', '/nl-NL/about']) {
+    for (const path of ['/hello', '/en/hello', '/es/hello', '/nl/about']) {
```

Comment:
`/hello` should be matched with any or no locale, whereas the `/nl/about` matcher was specified with `locale: false` (an undocumented Next.js feature 🙄) so should only be matched with that specific locale

Reply:
I believe that `nl-NL` was previously asserted on here because the [Next.js docs](https://nextjs.org/docs/pages/building-your-application/routing/internationalization#getting-started) are confusing and seem to indicate that `nl-NL` should fallback to `nl`, but that's only for the initial i18n redirect on the index page - the matchers shouldn't do this (or at least they don't when I run Next.js locally). The test appeared to pass, but that was in fact only due to the bug we're trying to fix here.

Reply:
But our i18n config - https://github.com/netlify/next-runtime/blob/a7620fdd9009f25fb7820dd43828a30aa22ff434/tests/fixtures/middleware-conditions/next.config.js#L5 doesn't match with Vercel's fixture - https://github.com/vercel/next.js/blob/canary/test/e2e/middleware-custom-matchers-i18n/app/next.config.js#L3 so probably this whole test was ~borked?